RubeesEat/Model/InMemory/InMemoryPersonRepository.cs

```diff
@@ -36,7 +38,7 @@ public Person GetCurrentUser()
         return _persons[CurrentUser];
     }
 
-    public Person? GetById(Guid guid)
+    public Person GetById(Guid guid)
```

Comment:
Why was this changed from Person? to Person
I thought we wanted to return null in case of failure, like in the other branches

Reply:
dont care

Reply:
wir mergen dann eh ;)